graveyard.json

```diff
@@ -1,4 +1,12 @@
 [
+  {
+    "dateClose": "2019-10-17",
+    "dateOpen": "2017-06-01",
+    "description": "Datally (formerly Triangle) was a smart app by Google that helped you save, manage, and share your mobile data.",
+    "link": "https://support.google.com/datally/?hl=en-GB#topic=7568779",
```

Comment:
Please use a different link. We should avoid using links directly from Google's sites because they tend to disappear. In level of priority:

1. Wikipedia
2. Reputable News Source
3. Other Sources (Including Google)

It is preferred that the link also mention the closing date.